build/teamcity/cockroach/ci/tests/check_generated_code_impl.sh

```diff
@@ -39,5 +39,8 @@ check_workspace_clean 'dev_generate' "Run \`./dev generate\` to automatically re
 # docs.
 bazel run @go_sdk//:bin/go --ui_event_filters=-DEBUG,-info,-stdout,-stderr --noshow_progress mod tidy
 check_workspace_clean 'go_mod_tidy' "Run \`go mod tidy\` to automatically regenerate these."
+# Run `generate-logictest` and ensure nothing changes.
+bazel run //pkg/cmd/generate-logictest -- -out-dir="$PWD"
+check_workspace_clean 'generate_logictest' "Run \`./dev gen logictest\` to automatically regenerate these."
```

Comment:
Nit: `dev gen testlogic` is the version we use in the code and is I think the "official" spelling, so maybe use it here. `./dev gen logictest` is a synonym so it still works.